.env.example

```
@@ -7,4 +7,8 @@ GOOGLE_MAPS_API_KEY=
 TMDB_API_KEY=
 TMDB_READ_ACCESS_TOKEN=
 GOOGLE_TRANSLATE_API_KEY=your_google_cloud_translation_api_key
-ELEVENLABS_API_KEY=your_elevenlabs_api_key
\ No newline at end of file
+ELEVENLABS_API_KEY=your_elevenlabs_api_key
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ELEVENLABS_API_KEY key should go before the GOOGLE_MAPS_API_KEY key

(UnorderedKey)